sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1089,6 +1145,20 @@ public void setEnableExternalConfiguration(boolean enableExternalConfiguration)
     Breadcrumb execute(@NotNull Breadcrumb breadcrumb, @Nullable Object hint);
   }
 
+  /** The traces sampler callback. */
+  public interface TracesSamplerCallback {
```

Comment:
all the other interfaces have its own .java class, I'd keep in the same file only if it's private/internal.

Reply:
`BeforeBreadcrumbCallback`, `BeforeSendCallback` are also a part of `SentryOptions` so i think this is consistent.

Reply:
true, my bad, guess I was sleepy :D or thinking about Integration, EventProcessor...